procsim/biomass/level0_product_generator.py

```diff
@@ -317,8 +324,10 @@ def generate_output(self):
         self._hdr.product_type = self._resolve_wildcard_product_type()
         self._hdr.incomplete_l0_slice = False
         self._hdr.partial_l0_slice = False
-        acq = self._hdr.acquisitions[0]
-        acq.slice_frame_nr = None
+        
+        # Get slice number from middle of slice to deal with merged slices.
+        middle_position = self._hdr.begin_position + (self._hdr.end_position - self._hdr.begin_position) / 2
+        self._hdr.acquisitions[0].slice_frame_nr = self._get_slice_frame_nr(middle_position, constants.SLICE_GRID_SPACING)
```

Comment:
I think this is wrong, according to the class description: "set wrsLatitudeGrid, aka the slice_frame_nr, to '___'".
I assumed that these merged products have no slice number. Or did I miss something?

Reply:
I think I was the one who missed something when I assumed these products were sliced. I've reverted it in https://github.com/stcorp/procsim/pull/17/commits/672e56460cacbfd00ebf94ab26b392ad0791c5a1.